src/Lecture1.hs

```diff
@@ -81,7 +84,11 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Integer -> Integer -> Integer -> Integer
+minmax x y z = difference
+    where difference = biggest - smallest
```

Comment:
In hindsight, using `difference` is a bit silly :facepalm:

Reply:
I agree, probably  you can write just `biggest - smallest` in the function result itself 🙂